src/nemo-devicelock/private/settingswatcher.cpp

```diff
@@ -277,6 +295,8 @@ void SettingsWatcher::reloadSettings()
     read(settings, this, "code_is_mandatory", false, &codeIsMandatory, &SettingsWatcher::codeIsMandatoryChanged);
     read(settings, this, "code_generation", AuthenticationInput::NoCodeGeneration,
          &codeGeneration, &SettingsWatcher::codeGenerationChanged);
+    read(settings, this, "temporary_lock_timeout", static_cast<qint64>(-1), &temporaryLockTimeout,
```

Comment:
I'd rather have a default value for this. Like 10 minutes that was talked about.

Reply:
"settingswatcher" -> this is middleware helper / qt bindings. What the default actually is depends on code and config files elsewhere. Using historical fallback value of -1 (for not-in-use) in non-authoritative source makes sense to me.

Yup, "duration" might have been better than "timeout". But this is about exposing already existing value rather than adding something brand new.